showcase/examples/history/history-extractor-script.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const util = require('util');
+// only runs on node 8 so be careful!
```

Comment:
is the error message explicit about it? If not might make sense to check for the running node version. `process.versions.node` if I'm not mistaken